R/project.R

```diff
@@ -33,6 +33,7 @@ project <- function(removals, fleet.props, dem_params, prev_naa, recruitment, op
     disc_caa_tmp    = array(NA, dim=c(1, model_params$nages, model_params$nsexes, model_params$nregions, model_params$nfleets))
     caa_tmp         = array(NA, dim=c(1, model_params$nages, model_params$nsexes, model_params$nregions, model_params$nfleets))
     faa_tmp         = array(NA, dim=c(1, model_params$nages, model_params$nsexes, model_params$nregions, model_params$nfleets))
+    F_f_tmp         = array(NA, dim=c(1,  model_params$nregions, model_params$nfleets))
```

Comment:
Always use the same dimensions as everything else, even if some of those dimension (e.g. age and sex in this case) seem unnecessary. Different dimensionality makes keeping track of things difficult.